docs/documentation/components/modal.html

```diff
@@ -152,15 +183,21 @@ <h3 class="title">Modal card</h3>
           </div>
           <nav class="level is-mobile">
             <div class="level-left">
-              <a class="level-item">
-                <span class="icon is-small"><i class="fas fa-reply"></i></span>
+              <a class="level-item" aria-label="reply">
```

Comment:
we should replace the <a> with <button> but if we did that now the style will be ugly, so I kept it.

I think maybe we rewrite some of the parts of the docs and this need time and effort or we can just write a comment above inaccessible component and say we should use x instead of y but we used y for styling purpose.